Linq2CouchBaseLiteExpression/Linq2CouchbaseLiteExpression.cs

```diff
@@ -188,6 +190,35 @@ private static Couchbase.Lite.Query.IExpression GenerateFromExpression(MethodCal
                         Couchbase.Lite.Query.Expression.Property(GetValueFromExpression(fieldName, null).ToString())
                                    .EqualTo(Couchbase.Lite.Query.Expression.Value(string.Empty)));
             }
+            else if (expression.Method.Name.Equals("Contains"))
+            {
+                var fieldName = expression.Arguments[0];
+                var fieldPropertyName = GetValueFromExpression(fieldName, null).ToString();
+
+                Couchbase.Lite.Query.IExpression sampleOr = null;
+
+                var value = GetValueFromExpression(expression.Object, null);
+                if (value is IEnumerable)
```

Comment:
You can update _if (value is IEnumerable values)_ and replace _(value as IEnumerable)_ by _values_ in the foreach to avoid casting twice